server/index.js

```diff
@@ -155,7 +157,7 @@ app.post("/login/createAccount", async (request, response) => {
 
 app.post("/login/authenticate", async (request, response) => {
     try {
-        var res = await loginService.authenticate(request.query.username, request.query.password);
+        var res = await loginService.authenticate(request.body.username, request.body.password);
```

Comment:
same as above: check with Jonathan if we're still using async/await

Reply:
He said that async/await is fine.